superset-frontend/src/components/Table/cell-renderers/ActionCell/index.tsx

```diff
@@ -103,7 +103,7 @@ function ActionMenu(props: ActionMenuProps) {
   };
 
   return (
-    <StyledMenu onClick={handleClick}>
+    <StyledMenu role="button" onClick={handleClick}>
```

Comment:
Same here. I think each item should have a button role instead of the Menu itself?